app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:

```

Comment:
It returns int you must specify type, which all child classes will return
the same problem with player_info